### DISCUSSION TITLE
add patch in NC AIO

### DISCUSSION THREAD

**Discussion by 4tler** [nextcloud/all-in-one#1434] [Questions] [answered]
Hello everyone, after restarting the NC AIO instance, an update was unfortunately carried out which leads to a known error. There is already a patch for this error. But how do I feed this into NC AIO? I couldn't find anything on this and it may be of interest to others.

in my case I would like to implement this patch https://github.com/nextcloud/server/pull/35218

**Comment by szaimen** [comment, marked as answer]
In this case applying the patch could look like this:
```
# Go into the container
sudo docker exec --user www-data -it nextcloud-aio-nextcloud bash
```
```
# Now inside the container
wget https://patch-diff.githubusercontent.com/raw/nextcloud/server/pull/35258.patch
git apply --check 35258.patch && git apply 35258.patch
exit
```

**Reply by 4tler** [reply]
thank you very much.

**Comment by ndemarco** [comment]
Has something changed since 2022 for this answer?

```
nick@nextcloud:/$ sudo docker exec --user www-data -it nextcloud-aio-nextcloud bash
bfc743e29477:/var/www/html$ patch
bash: patch: command not found
bfc743e29477:/var/www/html$
```

**Reply by szaimen** [reply]
the git command gets used here

**Reply by ndemarco** [reply]
🤦
